pkg/sql/sqlbase/validate_test.go

```diff
@@ -153,25 +153,19 @@ var validationMap = []struct {
 	{
 		obj: ColumnDescriptor{},
 		fieldMap: map[string]validationStatusInfo{
-			"Name": {status: thisFieldReferencesNoObjects},
-			"ID":   {status: thisFieldReferencesNoObjects},
-			"Type": {
-				status: todoIAmKnowinglyAddingTechDebt,
-				reason: "initial import: TODO(rohan): add validation for custom types"},
-			"Nullable": {status: thisFieldReferencesNoObjects},
-			"DefaultExpr": {
-				status: todoIAmKnowinglyAddingTechDebt,
-				reason: "initial import: TODO(features): add validation"},
-			"Hidden": {status: thisFieldReferencesNoObjects},
+			"Name":        {status: thisFieldReferencesNoObjects},
+			"ID":          {status: thisFieldReferencesNoObjects},
+			"Type":        {status: iSolemnlySwearThisFieldIsValidated},
+			"Nullable":    {status: thisFieldReferencesNoObjects},
+			"DefaultExpr": {status: iSolemnlySwearThisFieldIsValidated},
+			"Hidden":      {status: thisFieldReferencesNoObjects},
 			"UsesSequenceIds": {
 				status: todoIAmKnowinglyAddingTechDebt,
 				reason: "initial import: TODO(features): add validation"},
 			"OwnsSequenceIds": {
 				status: todoIAmKnowinglyAddingTechDebt,
 				reason: "initial import: TODO(features): add validation"},
-			"ComputeExpr": {
-				status: todoIAmKnowinglyAddingTechDebt,
-				reason: "initial import: TODO(features): add validation"},
+			"ComputeExpr": {status: iSolemnlySwearThisFieldIsValidated},
```

Comment:
I have detected a sworn oath violation - where is the validation for ComputeExpr in this PR? 

![image](https://user-images.githubusercontent.com/43821/86951110-13e2d080-c11f-11ea-9f13-909aa991b545.png)

Reply:
hehe the `desc.GetAllRefrencedTypeIDs` looks at `DefaultExpr`, `ComputeExpr` and `Predicate`, so it's covered. There's a test in this PR for it

Reply:
I should probably undo it though, we should ensure that all the columns referenced by those exprs exist, not just that the types they reference are valid

Reply:
Ah I see. Yes, we need to do more validation than just this. In fact, @mgartner and I just noticed a bug in `Predicate` that would be caught by a thorough `Validate` and not by this PR.

Reply:
removed.